frontend/app/src/types/settings/accounting.ts

```diff
@@ -57,3 +51,9 @@ export interface AccountingRuleRequestPayload
   readonly eventSubtypes?: string[];
   readonly counterparties?: string[];
 }
+
+export interface AccountingRuleLinkedSettingMap {
+  identifier: string;
+  label: string;
+  state: Ref<any>;
```

Comment:
Can we do something about this any here? the type should not be any right? can we infer the proper type?